resources/definitions/fdmextruder.def.json

```
@@ -108,6 +120,18 @@
                     "settable_per_meshgroup": false,
                     "type": "str"
                 },
+                "machine_extruder_start_code_duration":
+                {
+                    "default_value": 0,
+                    "description": "Duration it takes to execute the start g-code to when switching away from this extruder.",
```

```suggestion
                    "description": "The time it'll take to execute the start g-code, when switching to this extruder.",
```

Same as above, and the word 'to' landed in the wrong place I think.